datadog/tests/data_source_datadog_service_level_objective_test.go

```diff
@@ -0,0 +1,108 @@
+package test
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func TestAccDatadogServiceLevelObjectiveDatasource(t *testing.T) {
+	ctx, accProviders := testAccProviders(context.Background(), t)
+	uniq := strings.ReplaceAll(uniqueEntityName(ctx, t), "-", "_")
+	accProvider := testAccProvider(t, accProviders)
+
+	resource.ParallelTest(t, resource.TestCase{
```

Comment:
Sorry, you caught us in a time of some refactors/improvements to the project 🙂 
Could you update the beginning of these tests to match - https://github.com/DataDog/terraform-provider-datadog/blob/master/datadog/tests/data_source_datadog_dashboard_test.go#L13-L18